src/pages/Lend/FilterAlpha/LendFilterAlpha.vue

```diff
@@ -11,6 +11,41 @@
 			</div>
 			<div class="lg:tw-absolute lg:tw-w-full lg:tw-h-full lg:tw-top-0 lg:tw-pt-8">
 			</div>
+			<kv-page-container>
+				<h2 class="tw-mb-2">
+					Make a loan, change a life
+				</h2>
+				<h3 class="tw-mb-3">
+					Each Kiva loan helps people build a better future for themselves and their families.
+				</h3>
+				<kv-grid class="tw-grid-cols-2 md:tw-grid-cols-3">
+					<div class="tw-bg-gray-300 text-center sm:text-left"><p>Filters</p>
```

Comment:
This is minor and not required, but wanted to mention it. A common practice is to write responsive code "mobile first". What that means is that the default code is for mobile, and then you layer changes on top of that as the screen sizes increase.

So in this case, we want small screens to be text-left, and medium and up to be text-center. We can write that like
`text-left md:text-center` 
instead of 
`sm:text-left text-center`

Both of those will do the same thing! It's just a different way of thinking about how we layer styles.

--

Also, to get these classes to work in Kiva's application, they'll need to `tw` prefix. So `tw-text-left md:tw-text-center`